tests/unit/build/CMakeLists.txt

```diff
@@ -137,6 +137,20 @@ create_cmake_test(
   "${CMAKE_INSTALL_FULL_LIBDIR}/cmake/${HPX_PACKAGE_NAME}" MACROS
 )
 
+# Google test emulation
+create_cmake_test(
+  cmake_build_dir_gtest_emulation_test
+  "${PROJECT_BINARY_DIR}/lib/cmake/${HPX_PACKAGE_NAME}"
+  "examples/gtest_emulation" TARGETS
+)
+
+create_cmake_test(
+  cmake_install_dir_gtest_emulation_test
+  "${CMAKE_INSTALL_FULL_LIBDIR}/cmake/${HPX_PACKAGE_NAME}"
+  "examples/gtest_emulation" TARGETS
+)
```

Comment:
The `hpx_main` test can only work on Windows and I'm not sure the `hpx_start` test is useful (at least as it is). So we should disable the `hpx_main` test either here or in `examples/gtest_emulation/CMakeLists.txt`.